Tests/SwiftFusionTests/Datasets/G2OReader.swift

```diff
@@ -33,19 +35,50 @@ final class G2OReaderTests: XCTestCase {
     XCTAssertEqual(dataset, expectedDataset)
   }
 
+  /// Tests loading a simple 3d g2o file using G2OReader.
+  func testLoadSimple3D() {
+    /// Returns the expected `Pose3` that should be read from `g2oEntries`.
+    func pose3(_ g2oEntries: [Double]) -> Pose3 {
+      let s = g2oEntries
+      let t = Vector3(s[0], s[1], s[2])
+      let r = Rot3.fromQuaternion(s[6], s[3], s[4], s[5])
```

Comment:
w, x, y, z but g2o is x, y, z, w, thus above

future note: should make this function use labels `fromQuat(w:, x:, y:, z:)` to avoid typos